--- lxterminal-0.3.0/data/lxterminal.conf.in.old
+++ lxterminal-0.3.0/data/lxterminal.conf.in
@@ -1,4 +1,17 @@
 [general]
-fontname=Monospace 10
+fontname=Monospace 11
 selchars=-A-Za-z0-9,./?%&#:_
 scrollback=1000
+bgcolor=#000000000000
+bgalpha=65535
+fgcolor=#ffffffffffff
+disallowbold=false
+cursorblinks=false
+cursorunderline=false
+audiblebell=false
+tabpos=top
+hidescrollbar=false
+hidemenubar=false
+hideclosebutton=false
+disablef10=true
+disablealt=false
